src/components/TextField.vue

```diff
@@ -459,6 +460,12 @@ export default {
         @apply opacity-50 transition-opacity duration-150;
       }
     }
+
+    &.text-field--labelless {
+      .text-field__input {
+        @apply py-4;
+      }
+    }
```

Comment:
Can you move these definitions out of the scope they're in to reduce the level of nesting? In theory this should be enough on its own:

```scss
.text-field--labelless {
  .text-field__input {
    @apply py-4;
  }
}
```

Reply:
I tried that before but sadly that won't work without me setting those fields to important. I think this is cleaner for now.

Reply:
Yeah well that problem is only going to get worse if we keep doing this. Fine for now though.

Reply:
I'll open an issue to refactor the CSS for that c so we don't forget.